pipeline/data/download-mono.py

```diff
@@ -57,10 +54,10 @@ def main(args_list: Optional[list[str]] = None) -> None:
         "--max_sentences", type=int, help="The maximum number of sentences to retain"
     )
     parser.add_argument(
-        "--hlpt_min_fluency",
+        "--hlpt_min_doc_score",
```

Comment:
Just a minor thing. Maybe this PR could also fix the HPLT names? It is mistakenly written HLPT in here and in other parts of the code.